build.gradle

```diff
@@ -45,10 +45,15 @@ dependencies {
     compileOnly localGroovy()
     compileOnly gradleApi()
     compileOnly "org.codehaus.groovy:groovy:${groovyVersion}"
-    compileOnly "org.liquibase:liquibase-core:${liquibaseCoreVersion}"
-    // implementation "org.liquibase:liquibase-standard:${liquibaseCoreVersion}"
+    implementation "org.liquibase:liquibase-core:${liquibaseCoreVersion}"  // Changed from compileOnly
```

Comment:
This should not be necessary (we want to compile against, but don't want to include liquibase-core into this plugins code).

You can have it both as `compileOnly` and `testImplementation`. That should allow the test to run against liquibase-core code, without bringing liquibase-core into the plugin package.